--- origsrc/gurlchecker-0.10.2/src/ucclam.c	2007-04-25 00:41:44.000000000 -0500
+++ src/gurlchecker-0.10.2/src/ucclam.c	2008-07-06 16:17:08.500000000 -0500
@@ -34,9 +34,8 @@
   clam->limits.maxfiles = 1;
   clam->limits.maxfilesize = 10 * 1048576;
   clam->limits.maxreclevel = 1;
-  clam->limits.maxratio = 200;
   clam->limits.archivememlim = 0;
-  if (cl_loaddbdir (cl_retdbdir (), &clam->root, &clam->no) ||
+  if (cl_load (cl_retdbdir (), &clam->root, &clam->no, CL_DB_STDOPT) ||
       cl_build (clam->root))
     {
       uc_ucclam_free (clam);
